src/terragraph-e2e/lua/tg/frr_utils.lua

```diff
@@ -295,17 +306,32 @@ function frr_utils.fillConfigTemplate(
 
   -- Fill in neighbors
   local neighborsInfo, afNeighborsInfo, activeNeighbors = {}, {}, {}
+  local neighborMd5Password = ""
+  local neighborGeneralConfig = ""
```

Comment:
- delete the `= ""` assignment or else the linter (luacheck) complains about unused value
- declare these inside the loop instead (always declare at the minimum needed scope) - also you can just declare/assign these once instead of separately

Reply:
moved it into the loop